app/models/bool_symptom.rb

```diff
@@ -1,7 +1,12 @@
 # frozen_string_literal: true
 
 # BoolSymptom: a symptom that contains a bool
+# ActiveRecord will automatically typecast most types to Boolean.
+# Methods that create BoolSymptoms should keep the above in mind.
 class BoolSymptom < Symptom
+  validates :bool_value, inclusion: { in: [true, false] }
+  validates :bool_value, exclusion: { in: [nil] }
```

Comment:
bool_value needs to be able to be nil. In the case that the user sends an SMS or Voice response with "Yes", a symptomatic assessment is created where all of the symptom values are nil. This way the assessment shows up as symptomatic with blank specific symptoms which will be populated by the public health official who reaches out to them. See https://github.com/SaraAlert/SaraAlert/blob/b1ec8b77d70e30a32e5662185bd9a3bedcf5e271/app/models/threshold_condition.rb#L5 which is what generates the empty symptoms when an assessment comes in via SMS/Voice marked as symptomatic